website/docs/zh-CN/switch.md

```diff
@@ -143,7 +143,7 @@
 
 | 参数      | 说明    | 类型      | 可选值       | 默认值   |
 |---------- |-------- |---------- |-------------  |-------- |
-| value / v-model | 绑定值 | boolean / string / number | — | — |
+| value / v-model | 绑定值,必须等于active-value或inactive-value,默认为boolean类型 | boolean / string / number | — | — |
```

Comment:
Latin words and Chinese characters should have spaces directly, and Chinese punctuation should use full angles.Chinese punctuation should be full-angle.

Reply:
Agreed, for Japanese, use 。instead of .

Reply:
Also I do not think using translation service to do this is a good idea, translator sometimes translate to odd statement for native speakers